### PR TITLE
Result history in withdraw page

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-14 at 5 15 24 pm

File: src/languageProvider/locales/en_US.json [hunk 1]
```diff
@@ -137,12 +137,16 @@
   "str.token": "Token",
   "str.transaction": "TRANSACTIONS",
   "str.transactionId": "Transaction ID",
+  "str.resultHistory": "RESULT HISTORY",
```
Comment (dwalintukan) [inline src/languageProvider/locales/en_US.json]: use normal casing, and use the `i18nUtil.i18nToUpperCase()` method.

File: src/scenes/Event/components/EventTxHistory/resultHistory.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+class EventResultHistory extends React.PureComponent {
+  render() {
+    const { classes, oracles } = this.props;
+
+    const sortedOracles = _.orderBy(oracles, ['endTime']);
+
+    return (
+      <div className={classes.detailTxWrapper}>
+        <Typography variant="headline" className={classes.detailTxTitle}>
+          <FormattedMessage id="str.resultHistory" defaultMessage="RESULT HISTORY" />
```
Comment (dwalintukan) [inline src/scenes/Event/components/EventTxHistory/resultHistory.js]: example
```
<FormattedMessage id="eventInfo.fund" defaultMessage="Funding" >
    {(txt) => i18nToUpperCase(txt)}
</FormattedMessage>
```

File: src/scenes/Event/components/EventTxHistory/resultHistory.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                  <TableCell padding="dense">
+                    <FormattedMessage id="str.amount" defaultMessage="Amount" />
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {_.map(sortedOracles, (oracle, index) => (
+                  <TableRow key={`result-${index}`} selected={index % 2 === 1}>
+                    <TableCell padding="dense">{getLocalDateTimeString(oracle.endTime)}</TableCell>
+                    <TableCell padding="dense">{oracle.token === 'QTUM' ? 'QTUM Predict' : `Voting Round ${index}`}</TableCell>
```
Comment (dwalintukan) [inline src/scenes/Event/components/EventTxHistory/resultHistory.js]: 1. import constants for `Token.Qtum`
2. should we use `QTUM Betting` : `BOT Voting Round`?

File: src/scenes/Event/components/EventTxHistory/resultHistory.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            <Typography variant="body1">
+              <FormattedMessage id="str.emptyTxHistory" defaultMessage="You do not have any transactions right now." />
+            </Typography>
+        }
+      </div>
+    );
+  }
+}
+
+EventResultHistory.propTypes = {
```
Comment (dwalintukan) [inline src/scenes/Event/components/EventTxHistory/resultHistory.js]: please move to class as static type

File: src/scenes/Event/components/EventTxHistory/resultHistory.js (hunk 3 above)
Comment (liviavinci) [reply src/scenes/Event/components/EventTxHistory/resultHistory.js]: well bet/vote are actually actions, but here we are showing results, and we have the token name in the amount label already, so Betting Round : Voting Round?

Comment (dwalintukan) [reply src/scenes/Event/components/EventTxHistory/resultHistory.js]: @liviavinci Betting Round and Voting Round sounds good :)

**Comment by dwalintukan** [comment]
🎊

**Pull request merged**